SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -342,4 +345,14 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	// before the actor holding the position for all the hierarchy, it can immediately attempt to migrate back.
 	// Using this timestamp, we can back off attempting migrations for a while.
 	uint64 AuthorityReceivedTimestamp;
+
+	// Contains the frame at which the actor hierarchy was marked to be migrated to another worker.
+	// Used to delay the hierarchy's migration by one frame, to allow all actors in the hierarchy to go through
+	// the regular replication loop (to run Pre-Replication events for instance) before being migrated together.
+	uint32 MigrationFrame;
```

Comment:
I think there might be a tiny bit of ambiguity for someone debugging since this is actually the frame when migration was initiated, rather than the actual migration frame as it literally implies. I know the comment clarifies, but can we also change the name to something like `FrameWhenActorHierarchyMigrationStarted`